src/eventLoopStats.cc

```diff
@@ -6,46 +6,72 @@ const uint32_t maxPossibleNumber = 4294967295;
 const uint32_t minPossibleNumber = 0;
 
 uv_check_t check_handle;
+
+// The minimum event loop duration since the last sense() call.
 uint32_t min;
+// The maximum event loop duration since the last sense() call.
 uint32_t max;
-uint32_t num;
+// The sum of event loop durations the last sense() call.
 uint32_t sum;
+// The number of event loop iterations since the last sense() call.
+uint32_t num;
```

Comment:
Just resorted those a little, because I feel sum has more to do with min and max than num has.